src/mlx_omni_server/rerank/rerank_service.py

```
@@ -0,0 +1,196 @@
+                    relevance_score=result["relevance_score"],
+                    document=result["document"] if request.return_documents else None,
+                )
+                rerank_results.append(rerank_result)
+
+            response = RerankResponse(
+                model=model_id,
+                results=rerank_results,
+                usage=RerankUsage(total_tokens=token_count),
+            )
+
+            return response
```

Ruff (0.14.7)

Consider moving this statement to an `else` block

(TRY300)